architecture.md

```diff
@@ -1,20 +1,82 @@
 # Architecture
-This section seeks explain some points about clean architecture. 
+This section seeks explain a little more about clean architecture.
+
+<br/>
+
+<img style="margin-bottom: 10px" src="architecture.png"/>
+
+<br/>
+
+## Summary
+This image is a summary of concepts, relationships and responsibilities of the existenting layers.
+With it in mind let's talk about the following topics:
+
+- [Purpose](#Purpose)
+- [Principles](#Principles)
+- [Concepts](#Concepts)
+- [Layers](#Layers)
+- [Relationships](#Relationships)
+- [Rules](#Rules)
+- [More about](#Moreabout)
+
+<br/>
+
+## Purpose 
+How objective this architecture seek solve or improve the following points:
+
+- structure expressivity
+- maintenance 
+- testability
+- decoupling of business solve from Frameworks and drives
+
+<br/>
 
 ## Principles
+The principles that guide this architecture are:
 
+- SoC (separation of concerns)
+- Use cases how first class citizens
+- Dependency injection
+- Testability
 
-## Rules
-## Layers
-## Request flow
-## Relationship bettwen layers
-## Implementation
-## More about
+<br/>
+
+## Concepts
+There are some concepts that are important we knowing before we following analyzing in the next sections.
+
+### Policies
+Software systems are statements of politics. In essence this mean what a program of computer really is. A program of computer is a description detail of a political that coordinates the transformations of input and output.
+
+#### High level
+How much more away of inputs and outputs more high is politic's level. Generally this type of politic describe flows more abstracts. It tend change with less frequency but for reasons more substantial.\
+Example: Changes in calculate of interest of enterprise.
+
+#### Low level
+The policies that cope with inputs and outputs from system are policies of more low level of the system. These policies tend to change more frequently for less substantial reasons because they are closer to I / O.\
```

Comment:
```suggestion
The policies that deal with inputs and outputs from system are policies of more low level of the system. These policies tend to change more frequently for less substantial reasons because they are closer to I / O.\
```